model_analyzer/config/generate/base_model_config_generator.py

```diff
@@ -109,14 +113,20 @@ def _make_direct_mode_model_config(self, param_combo):
         apply the changes on top of the base and return the new model config
         """
         model_config_dict = self._get_base_model_config_dict()
+        model_config_dict['name'] = self._get_model_variant_name(param_combo)
+        logger.info("")
+        logger.info(f"Creating model config: {model_config_dict['name']}")
+
         if param_combo is not None:
             for key, value in param_combo.items():
                 if value is not None:
                     model_config_dict[key] = value
-
-        model_config_dict['name'] = self._get_model_variant_name(param_combo)
+                    value_str = "ON" if value == {} else value
+                    logger.info(f"  Setting {key} to {value_str}")
```

Comment:
A small nit, but instead of saying "Setting {key} to ON", you could do something like:
if value == {}:
    logger.info(f"  Enabling {key}")
else:
   logger.info(f"  Setting {key} to {value_str}")